doc/classes/ProjectSettings.xml

```diff
@@ -970,6 +970,12 @@
 			Fix to improve physics jitter, specially on monitors where refresh rate is different than the physics FPS.
 			[b]Note:[/b] This property is only read when the project starts. To change the physics FPS at runtime, set [member Engine.physics_jitter_fix] instead.
 		</member>
+		<member name="rendering/batching/precision/uv_contract" type="bool" setter="" getter="" default="false">
+		On some platforms (especially mobile), precision issues in shaders can lead to reading 1 texel outside of bounds, particularly where rects are scaled. This can particularly lead to border artifacts around tiles in tilemaps. This adjustments corrects for this by making a small contraction to the UV coordinates used. Note that this can result in a slight squashing of border texels.
```

Comment:
Indentation issue (missing one tab).

Reply:
Maybe add a mention to `[member rendering/quality/2d/use_pixel_snap]` and mention this new setting in `use_pixel_snap`'s description, as I guess they're related and will often be needed together?

Reply:
Good catch, yup I'll add something. :+1: